### PR TITLE
Service reference landing page: IoT

### PR DESCRIPTION
**Pull request by dbradish-microsoft**

### CODE REVIEW

File: docs-ref-conceptual/azure-cli-reference-for-IoT.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ms.devlang: azurecli
+---
+
+# Azure CLI for Azure IoT
+
+The Azure Command Line Interface ([Azure CLI](/cli/azure/what-is-azure-cli)) is a set of commands used to create and manage Azure resources.  It is available across many Azure services including Azure IoT.  There are over 100 references for Azure IoT giving you the ability to work effectively with IoT services from a command line.
+
+## References for IoT
+
+There are two type of Azure CLI references for IoT: released and extension.  Released references have been published as a permanent part of the Azure CLI.  Extension references haven't shipped as part of the CLI but run as CLI commands. With extensions, you gain access to experimental and pre-release commands.
```
Comment (digimaun) [inline docs-ref-conceptual/azure-cli-reference-for-IoT.md:20]: For IoT, the CLI IoT extension should be treated and referred to like a flagship tooling product from Azure IoT. It does have experimental and pre-release commands, but significant quantities of the IoT CLI extension are not in this form and instead in long-term use form.

Put another way, the IoT extension is a natural half of the IoT CLI experience and IoT CLI users and IoT automation scenarios will not get far without it. In fact, all the data-plane functionality of IoT exists in the IoT extension.

File: docs-ref-conceptual/azure-cli-reference-for-IoT.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+The Azure Command Line Interface ([Azure CLI](/cli/azure/what-is-azure-cli)) is a set of commands used to create and manage Azure resources.  It is available across many Azure services including Azure IoT.  There are over 100 references for Azure IoT giving you the ability to work effectively with IoT services from a command line.
+
+## References for IoT
+
+The Azure IoT CLI experience is composed of two parts: Azure CLI (commonly referred to as CLI **core**) and the Azure IoT CLI **extension**.
+
+IoT functionality in Azure CLI **core** is focused on infrastructure management and configuration. IoT Hub CRUD operations, or configuring IoT Hub message routes are typical use cases for core commands.
+
+The IoT **extension** introduces rich features and functionality.  Extensions manage, manipulate and interact with the data, entities and objects on the infrastructure itself. Managing fleets of devices, monitoring device-to-cloud events and invoking cloud to device methods are all enabled via the IoT extension. Azure CLI extensions for IoT unlock the use of experimental or pre-release technology contributing to its versatility in a variety of scenarios and use cases.
```
Comment (digimaun) [inline docs-ref-conceptual/azure-cli-reference-for-IoT.md:24]: The second sentence where it starts with ". Extensions manage ..." - this description is specific to the IoT extension and there is only 1 certified IoT extension by Azure IoT. Also these are examples of functional categories of what the IoT extension can do, not an exhaustive list, which is why I think we need a "For example" disclaimer on there.

Refactor suggestion:
`
The IoT **extension** introduces rich features and functionality to manage, manipulate and interact with the data, entities and objects on the infrastructure itself. For example managing fleets of devices, monitoring device-to-cloud events and invoking cloud to device methods are all enabled via the IoT extension. The Azure IoT extension for Azure CLI unlocks the use of experimental or pre-release technology contributing to its versatility in a variety of scenarios and use cases.
`

File: docs-ref-conceptual/azure-cli-reference-for-IoT.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+### Links to core and extension commands
+
+| Reference | Core | Extension | Description
+|-|-|-|-|
+| az iot | [core](/cli/azure/iot) | [extension](/cli/azure/ext/azure-iot/iot)  | All available Azure IoT commands.
+| az iot central | [core](/cli/azure/iot/central) | [extension](/cli/azure/ext/azure-iot/iot/central) | Manage IoT Central assets.
+| az iot device | | [extension only](/cli/azure/ext/azure-iot/iot/device) | Leverage device-to-cloud and cloud-to-device messaging capabilities.
+| az dt | | [extension only](/cli/azure/ext/azure-iot/dt) | Manage Azure Digital Twins (ADT) solutions & infrastructure.
```
Comment (digimaun) [inline docs-ref-conceptual/azure-cli-reference-for-IoT.md:33]: Let's please remove the `(ADT)` part of the description.

**Comment by dbradish-microsoft** [comment]
@digimaun , final version?

**Pull request merged**